app/presenters/tree_node/node.rb

```diff
@@ -78,6 +78,8 @@ def escape(string)
     end
 
     def to_h
+      cursor_class = selectable == false ? 'no-cursor' : ''
```

Comment:
This could be just
```ruby
cursor_class = selectable ? '' : 'no-cursor'
```
Or can `selectable` be a non-bolean?

Reply:
No, you're right! My assumption was that the `selectable` doesn't have a default value, but now I see but it has `true`. This case was to prevent `nil` values from adding the `no-cursor`, but that can never happen...fixing.